astroplan/core.py

```diff
@@ -161,6 +163,46 @@ def altaz(self, time, target=None, obswl=None):
                 coordinate = target
             return coordinate.transform_to(altaz_frame)
 
+    def parallactic_angle(self, time, target):
```

Comment:
Should this function be a module-level function called `parallactic_angle` also?